docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -0,0 +1,66 @@
+# Using devices with Clear Containers
+
+Host devices can be passed to a container with Docker using --device option
+with docker run. An example of passing a device with docker is given below:
+```
+docker run -it --device=/dev/sdb debian
+```
+Clear Containers has support for passing vfio device groups and block
+devices to the container with --device.
+
+## VFIO Devices (PCI Passthrough)
+
+A device can be directly passed to a Clear Containers container using 
+Virtual Function I/O (VFIO). The VFIO driver is an IOMMU/device
+agnostic framework for exposing direct device access to userspace, in
+a secure, IOMMU protected environment. 
+
+For information on VFIO, please refer to:
+ - https://www.kernel.org/doc/Documentation/vfio.txt
+ - https://www.linux-kvm.org/images/b/b4/2012-forum-VFIO.pdf
+
+A device can pe passed to a Clear Container container using VFIO
```

Comment:
Can I pass multiple `--device`'s? If so, I'd change it to, "Devices can be passed to...".